Makefile

```diff
@@ -86,6 +86,12 @@ image: yamls
 	    $(foreach tag,$(IMAGE_EXTRA_TAGS),-t $(tag)-minimal) \
 	    $(IMAGE_BUILD_EXTRA_OPTS) ./
 
+# clean NFD labels on all nodes
+# devel only
+clean-labels:
+	kubectl get no -o yaml | sed -e '/^\s*nfd.node.kubernetes.io/d' -e '/^\s*feature.node.kubernetes.io/d' | kubectl replace -f -
+	kubectl apply -k deployment/overlays/prune/
```

Comment:
Hmm, need to think about this. We should probably do a `kubectl wait` on the completion of the job, or smth, and then do `k delete -k deployment/overlays/prune`. Otherwise subsequent `make clean-labels` don't have an effect (no new prune jobs will be created)

Reply:
how about now?